.github/workflows/csharpier.yml

```
@@ -29,8 +29,11 @@ env:
 jobs:
 
   csharpier:
-    permissions: write-all
     runs-on: ubuntu-latest
+        permissions:
```

actionlint (1.7.4)

could not parse as YAML: yaml: line 33: mapping values are not allowed in this context

(syntax-check)